list.js

```diff
@@ -23,14 +23,25 @@ class List {
     let newDiv = document.createElement('div')
     newDiv.setAttribute("id", `list_${list.id}`)
     newDiv.setAttribute('class', 'list')
+    newDiv.setAttribute('data-id', list.id)
+
+    let button = document.createElement('button')
+    button.innerText = "X"
+
     let title = document.createElement('h2')
-    title.innerText = list.title
+    title.append(button)
+    title.append(` ${list.title}`)
+
     newDiv.appendChild(title)
+
     let elementList = document.createElement('ul')
     elementList.setAttribute('class', 'task')
     newDiv.appendChild(elementList)
+
     let listsDiv = document.getElementById('lists')
     listsDiv.appendChild(newDiv)
+
+    this.deleteListButtonListener(button)
```

Comment:
So, after the creation of the list, we need to add a listener to the `delete` button.